sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -3023,7 +3020,8 @@ private <DestinationT> WriteResult continueExpandTyped(
                 getStorageApiTriggeringFrequency(bqOptions),
                 getBigQueryServices(),
                 getStorageApiNumStreams(bqOptions),
-                method == Method.STORAGE_API_AT_LEAST_ONCE);
+                method == Method.STORAGE_API_AT_LEAST_ONCE,
+                getAutoSharding());
```

Comment:
Does autosharding default to true or false?

Reply:
The Write method sets autosharding as false by default: https://github.com/apache/beam/blob/0fbecde63b3b3f89ab7590f62e9206610767b66c/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java#L1768. The user has to explicitly enable it